site/components/image/index.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+
+type Props = {
+  alt?: string,
+  className?: string,
+  role?: string,
+  src: string,
+  src2x?: string,
+  src3x?: string,
+};
+
+const Image = ({ alt, className, role, src, src2x, src3x }: Props) => {
+  const src2xTemplate = src2x ? `, ${src2x} 2x` : '';
+  const src3xTemplate = src3x ? `, ${src3x} 3x` : '';
+  return (
+    <picture>
+      {src && <source srcSet={`${src}${src2xTemplate}${src3xTemplate}`} className={className} />}
+      <img src={src} alt={alt || ''} role={role} className={className} />
+    </picture>
+  );
+};
+
+Image.defaultProps = {
+  alt: '',
+  className: undefined,
+  role: undefined,
+  src: undefined,
```

Comment:
I don't think we are doing the right thing here. you are providing `alt` as default but then also have an `||` in the code. You are providing `undefined` as defaults but these will be displayed as `undefined` in the HTML. Maybe you are misunderstanding how optional parameters work in flow https://flow.org/en/docs/types/functions/#toc-optional-parameters ?

Reply:
updated